sklearn/impute/_knn.py

```diff
@@ -72,6 +72,12 @@ class KNNImputer(_BaseImputer):
         missing indicator even if there are missing values at transform/test
         time.
 
+    keep_empty_features : bool, default=False
+        If True, features that consist exclusively of missing values `fit` is
+        called not removed when calling `transform`. The imputed value is `0`.
```

Comment:
Ditto: I think there is a small syntax mistake. Also, is using this affirmative phrasing better?
```suggestion
        If True, features that consist exclusively of missing values when
        `fit` is called are returned in results when `transform` is called.
        The imputed value is always `0`.
```